### PR TITLE
fix(doc): Replace iconography for position's email status with text label

### LINKED ISSUE #3855: IESG evaluation record: can't distinguish mail/no-mail icons
### What happened?

In an IESG evaluation record, there are small envelope-shaped icons in each ballot header that indicate whether mail is requested or not.

These icons are practically indistinguishable.  There are also three different ones, which two different tooltips.

https://datatracker.ietf.org/doc/draft-ietf-cbor-file-magic/ballot/

### What browser(s) are you seeing the problem on?

Chrome / Edge, Safari

### Code of Conduct

- [X] I agree to follow the IETF's Code of Conduct

### PR DESCRIPTION
**Pull request by aph3rson**
Positions within a ballot, discuss, or comment within DT may-or-may-not have an email notification associated with them. This is at the discretion of the submitter, and is indicated within DT to viewers of the thread.

Previously, this was indicated via an icon+tooltip on the specific position, using an icon which was less-than-readable (especially on low-DPI/small displays).

This change replaces that iconography with a text pill, while retaining the previous tooltip for additional information.

Closes #3855.

### CODE REVIEW

File: ietf/doc/tests_ballot.py [hunk 1]
```diff
@@ -1231,8 +1231,10 @@ def _assertBallotMessage(self, q, balloter, expected):
         heading = q(f'div.h5[id$="_{slugify(balloter.plain_name())}"]')
         self.assertEqual(len(heading), 1)
         # <div.h5> is followed by a panel with the message of interest, so use next()
+        next = heading.next()
+        print(f"looking for {expected} in:\n{next}")
```
Comment (rjsparks) [inline ietf/doc/tests_ballot.py:1235]: Leftover debugging?

**Review by rjsparks — CHANGES_REQUESTED** [review]

File: ietf/doc/tests_ballot.py (hunk 1 above)
Comment (aph3rson) [reply ietf/doc/tests_ballot.py]: Fixed in b430fa2583446581c2de942618b1efba5cb496fd.

**Comment by aph3rson** [comment]
@rjsparks any thoughts on this?

**Comment by rjsparks** [comment]
Been needing time to review - meeting startup has been intense.
There are some things to change, but I'll pick them up later.

**Review by rjsparks — APPROVED** [review]

**Pull request merged**